### DISCUSSION TITLE
Lecture Naming

### DISCUSSION THREAD

**Discussion by jmgirard** [PittMethods/appliedml#14] [Ideas]
@ShirleyBWang Instead of Day 1 AM1, Day 1 AM2, etc. I think Day 1a, 1b, 1c, and 1d might be cleaner. What do you think?

**Comment by ShirleyBWang** [comment]
Yeah, I like that a lot. Can make that change! Sorry for slow reply - have
been traveling and back on 7/8. Hope things are good with you!!

On Thu, Jun 30, 2022 at 5:24 PM Jeffrey Girard ***@***.***>
wrote:

> @ShirleyBWang <https://github.com/ShirleyBWang> Instead of Day 1 AM1, Day
> 1 AM2, etc. I think Day 1a, 1b, 1c, and 1d might be cleaner. What do you
> think?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/PittMethods/appliedml/discussions/14>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AK3P4WH66MD64FAIJ5WBMHTVRWU3RANCNFSM52JRX53A>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Shirley B. Wang, A.M.
Ph.D. Candidate in Clinical Science
Secondary in Computational Science & Engineering
NIH F31 & NSF Graduate Research Fellow
Department of Psychology, Harvard University
shirleywang.rbind.io
She/Her/Hers
